test/remote/ups_test.rb

```diff
@@ -500,4 +502,23 @@ def test_create_shipment_with_insured_value
     assert response.success?
     assert_instance_of ActiveShipping::LabelResponse, response
   end
+
+  private
+
+  def with_ups_holidays
```

Comment:
This is the right idea! Should be extracted into something more generic and part of the test env setup. Used with other carriers. But good to get this PR green. Maybe cut an issue to make sure we don't forget to generalize this.